Unity-Technologies/ScriptableRenderPipeline
# Added support to Screen Space UI camera type.

### Purpose of this PR
 Camera Stacking solution applies post-processing at the end of the stack. This caused an undesirable behavior of users wanting to composite UI with an overlay camera.
 This PR exposes another camera render type `Screen Space UI` to render UI after Post-processing. 
 This is a very lean rendering camera that focuses on rendering UI only. (no occlusion culling, no lighting culling, no depth or color copy or shadow map)

---
### Testing status

**Manual Tests**: What did you do?
- [x ] Opened test project + Run graphic tests locally
- [x ] Built a player
- [x ] Checked new UI names with UX convention
- [x ] C# and shader warnings (supress shader cache to see them)
- Other: 

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

---
### Comments to reviewers

![UICamera](https://user-images.githubusercontent.com/7453395/72601651-202b6500-3916-11ea-9603-ff30b656d6d4.JPG)
